src/main/java/de/tum/in/www1/artemis/web/rest/tutorialgroups/TutorialGroupResource.java

```diff
@@ -531,6 +539,57 @@ private void checkEntityIdMatchesPathIds(TutorialGroup tutorialGroup, Optional<L
         });
     }
 
+    /**
+     * GET /courses/:courseId/tutorial-groups/export : Export tutorial groups for a specific course to a CSV file.
+     *
+     * @param courseId the id of the course for which the tutorial groups should be exported
+     * @param fields   the list of fields to include in the CSV export
+     * @return the ResponseEntity with status 200 (OK) and the CSV file containing the tutorial groups
+     */
+    @GetMapping(value = "courses/{courseId}/tutorial-groups/export/csv", produces = "text/csv")
+    @EnforceAtLeastInstructorInCourse
+    @FeatureToggle(Feature.TutorialGroups)
+    public ResponseEntity<byte[]> exportTutorialGroupsToCSV(@PathVariable Long courseId, @RequestParam List<String> fields) {
+        var course = courseRepository.findByIdElseThrow(courseId);
```

Comment:
Why do you not log this endpoint with the logger?

Reply:
added the log 👍